spec/lib/slack/block_kit/layout/section_spec.rb

```diff
@@ -187,4 +187,67 @@
       expect(section_json).to eq(expected_json.merge(accessory: expected_accessory_json))
     end
   end
+
+  context 'with fields and text' do
+    describe '#plaintext_field' do
+      let(:expected_plaintext_field_json) do
+        {
+          emoji: false,
+          text: '__FIELD_TEXT__',
+          type: 'plain_text'
+        }
+      end
+
+      it 'correctly serializes' do
+        instance.plaintext_field(text: '__FIELD_TEXT__', emoji: false)
+
+        expect(section_json).to eq(expected_json.merge(fields: [expected_plaintext_field_json]))
+      end
+    end
+
+    describe '#mrkdwn_field' do
+      let(:expected_mrkdwn_field_json) do
+        {
+          verbatim: false,
+          text: '__FIELD_MRKDWN__',
+          type: 'mrkdwn'
+        }
+      end
+
+      it 'correctly serializes' do
+        instance.mrkdwn_field(text: '__FIELD_MRKDWN__', verbatim: false)
+
+        expect(section_json).to eq(expected_json.merge(fields: [expected_mrkdwn_field_json]))
+      end
+    end
+  end
+
+  context 'with fields but no text' do
+    let(:instance) do
+      block = described_class.new(block_id: '__BLOCK__')
+      block
+    end
```

Comment:
```suggestion
    let(:instance) { described_class.new(block_id: '__BLOCK__') }
```

This can be inlined!